.github/workflows/deploy_production.yml

```
@@ -0,0 +1,25 @@
+name: "Deploy production"
+on: [ workflow_dispatch ]
+
+env:
+  CI: "true"
+  SIMPLECOV: "true"
+  RUBY_VERSION: 2.7.5
+  RAILS_ENV: test
+  NODE_VERSION: 16.9.1
+  RUBYOPT: '-W:no-deprecated'
+
+jobs:
+  deploy_master:
+    if: "github.ref == 'refs/heads/master'"
+    name: Deploy master branch on production instance
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run Ansible playbook
+        uses: appleboy/ssh-action@v0.1.4
+        with:
+          host: ${{ secrets.ANSIBLE_HOST }}
+          username: ${{ secrets.ANSIBLE_USERNAME }}
+          key: ${{ secrets.ANSIBLE_KEY }}
+          port: ${{ secrets.SSH_PORT }}
+          script: ansible-playbook -u ${{ secrets.ANSIBLE_USERNAME }} --private-key="~/.ssh/ansible-deploy/ansible-deploy" -i /home/${{ secrets.ANSIBLE_USERNAME }}/ansible/decidim/inventories/production.yml /home/${{ secrets.ANSIBLE_USERNAME }}/ansible/decidim/playbooks/update_decidim_app.yml
```

```suggestion
          /home/${{ secrets.ANSIBLE_USERNAME }}/ansible/decidim/inventories/cese.yml 
```

We probably want to create a new inventory named `cese.yml` to prevent updating all instances on push for CESE

---

Suggestions is just to show where is the inventory file name, but we should keep the same command